src/applications/facility-locator/components/facility-details/OperationStatus.jsx

```diff
@@ -48,7 +48,7 @@ export default function OperationStatus(props) {
   return (
     <va-alert close-btn-aria-label="" status={alertClass} visible>
       {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
-      <h2 slot="headline" tabIndex={0} role="alert">
+      <h2 slot="headline" role="alert">
```

Comment:
**Note:** This is an incorrect use of `role="alert"` but that's something that should be investigated and documented in another ticket.